tests-cuda/v2/test_1276-cuda-num.py

```diff
@@ -40,7 +40,7 @@ def test_num_3():
         ak.contents.numpyarray.NumpyArray(np.array([0.0, 1.1, 2.2, 3.3, 4.4, 5.5]))
     )
     cuda_array = ak.to_backend(array, "cuda")
-    ak.num(cuda_array, 0)
+    assert ak.num(cuda_array, 0) == ak.num(array, 0)
```

Comment:
Although Doug in dask-awkward is using `import awkward._v2 as ak`, we shouldn't use it in the Awkward codebase because we don't want one codebase to mix both `import awkward._v2 as ak` and `import awkward as ak`. That would get confusing!

I was about to say, "this should be `ak._v2.num`, rather than `ak.num`," until I noticed how it was imported.